README.md

```diff
@@ -10,6 +10,31 @@ and self-esteem by navigating the life of their avatar!
 1. Clone the repo to your desktop `git clone git@github.com:your_name/powerup-android.git`
 1. Open the project with Android Studio 
 
+## Configure remotes
+When a repository is cloned, it has a default remote called `origin` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named `upstream`:
+
+Open terminal or git bash in your local repository and type:
+
+`git remote add upstream https://github.com/systers/powerup-android.git`
+
+Run `git remote -v` to check the status, you should see something like the following:
+
+origin    https://github.com/YOUR_USERNAME/powerup-android.git (fetch)
```

Comment:
Can you add these four lines inside a code snipet so it's clear that it's not something you write but something you see?